avocado/plugins/runner_nrunner.py

```diff
@@ -19,23 +19,20 @@
 from avocado.core import test
 from avocado.core.plugin_interfaces import Runner as RunnerInterface
 
-from avocado.core.nrunner import check_tasks_requirements
+from avocado.core import nrunner
```

Comment:
Currently, we are using only `check_tasks_requirements` and `RUNNERS_REGISTRY_PYTHON_CLASS`. Any reason to import the entire module here?

Reply:
Nothing really extraordinary... Best explanation is this being part of my WiP, which is going to use more symbols from the nrunner module.